sql/core/src/main/scala/org/apache/spark/sql/execution/ui/ExecutionPage.scala

```diff
@@ -71,6 +71,10 @@ class ExecutionPage(parent: SQLTab) extends WebUIPage("execution") with Logging
             {jobLinks(JobExecutionStatus.FAILED, "Failed Jobs:")}
           </ul>
         </div>
+        <div>
+          <input type="checkbox" id="stageId-and-taskId-checkbox"></input>
+          <span>Show Stage/Task IDs</span>
```

Comment:
how about 
```
Show Stage/Task IDs with the max metrics
```

Reply:
Maybe we should just have this be the entire format:
Show the Stage (Stage Attempt): Task ID that corresponds to the max metric